app/src/lib/sanitize-branch.ts

```diff
@@ -6,7 +6,7 @@ const invalidCharacterRegex = /[\x00-\x20\x7F~^:?*\[\\|""<>]+|@{|\.\.+|^\.|\.$|\
 
 /** Sanitize a proposed branch name by replacing illegal characters. */
 export function sanitizedBranchName(name: string): string {
-  return name.replace(invalidCharacterRegex, '-').replace(/^-/g, '')
+  return name.replace(invalidCharacterRegex, '-').replace(/^[-\+]/g, '')
```

Comment:
Should we also at the 0..N modifier `*`? Otherwise, `++but-can-still-keep-the-rest` would be a valid branch name.

Reply:
Great catch, I will update this now! Thank you for the feedback.

Reply:
Well, with the `*` it should be able to detect several + and - symbols at the start. Should I combine the + and - tests into a single test?